### PR TITLE
Bbc 6764 create lock icon

### PR DESCRIPTION
**Pull request by nickoola**

### CODE REVIEW

**Comment by nickoola** [comment]
Capture d’écran 2020-02-14 à 18 43 23

**Comment by pith** [comment]
Here is what we expect in the specs 
image

**Comment by nickoola** [comment]
Here is the render after using the proper icon:
Capture d’écran 2020-02-17 à 14 46 11

File: src/datePicker/__snapshots__/DatePicker.unit.tsx.snap [hunk 1]
```diff
@@ -40,7 +40,7 @@ exports[`DatePicker renderNavbar Should render the previous/next buttons in hori
   display: -webkit-inline-flex;
   display: -ms-inline-flexbox;
   display: inline-flex;
-  padding: 16px 24px;
+  padding: 0 24px;
```
Comment (gabrielagabriel) [inline src/datePicker/__snapshots__/DatePicker.unit.tsx.snap]: **[no magic numbers]**  Setting paddings according to brand spaces, eg: `space.x` to inforce typography grid

Comment (Tuxosaurus) [reply src/datePicker/__snapshots__/DatePicker.unit.tsx.snap]: This is in the snapshot, it's fine in index.tsx :)

File: src/button/index.tsx [hunk 2]
```diff
@@ -19,21 +19,26 @@ const StyledButton = styled(Button)`
...
   }
 
   & svg,
   & img {
     flex-shrink: 0;
   }
 
+  &:not(.kirk-button-bubble) svg {
+    margin-right: ${space.s};
+  }
```
Comment (wakooka) [inline src/button/index.tsx]: I don't think we should do this :/ having exceptions inside a button component based on what icon is inside is defeating the purpose of normalizing styles.

**Comment by Bidjii** [comment]
@loick for review plz, he is the one concerned 😬

**Comment by loick** [comment]
Can you stop doing typo on my name please?

**Comment by nickoola** [comment]
Here is an update ()visually validated by @wakooka  using our brand spacer (Pixar specs are not precise for now in this case):
Capture d’écran 2020-02-26 à 13 22 59

File: src/button/index.tsx (hunk 2 above)
Comment (nickoola) [reply src/button/index.tsx]: I inverted the css logic ;-)

**Pull request merged**